.github/workflows/preview-documentation.yml

```diff
@@ -56,12 +56,11 @@ jobs:
           keep_files: false
 
       - name: Add comment about preview URL
-        uses: unsplash/comment-on-pr@v1.3.0
+        uses: unsplash/comment-on-pr@v1.2.0
```

Comment:
Is there any reason for choosing version 1.2 instead of `master`, as we had before?

Reply:
Because that is where the bug is 😂